yes.rs

```diff
@@ -1,19 +1,11 @@
 use std::env;
 
-#[allow(while_true)]
 fn main() {
     let args: Vec<String> = env::args().collect();
+    let args = &args[1..];
 
-    match args.len() {
-        1 => {
-            while true {
-                println!("y");
-            }
-        },
-        _ => {
-            while true {
-                println!("{}", args[1]);
-            }
-        },
+    let output = if args.is_empty() { "y".to_string() } else { args.join(" ") };
```

Comment:
I think it's necessary.
[Here](http://hermanradtke.com/2015/05/03/string-vs-str-in-rust-functions.html)
It's a Rust thing :grinning: